chia/rpc/util.py

```diff
@@ -43,8 +43,13 @@ async def rpc_endpoint(self, request: Dict[str, Any], *args: object, **kwargs: o
             *args,
             **kwargs,
         )
-        with clvm_serialization_mode(not request.get("full_jsonify", False)):
+        if request.get("full_jsonify", False):
```

Comment:
We're changing the default behavior to this new form?

Reply:
Yeah but it doesn't actually change anything for existing RPCs since none of them use clvm_streamables.  So you could also think of it as _clarifying_ a default that was implicit we just didn't know it :)

Reply:
We would expect to try to get consistency across our RPCs right?  Does that plan include breaking all of the existing RPCs?  I mean sure, not immediately, but talking about where we would like to be.

Reply:
Sorry, I think perhaps you misunderstand.  When clvm streamable is not involved, full_jsonify being true or false doesn't affect anything.  It's only specifying whether the clvm streamables should come across as fully json or with the relevant bits converted to hex blobs.   I'll give it a better name in this PR too to avoid further confusion.

Reply:
(clvm streamables are not a part of any existing RPCs)

Reply:
Changed to reference the specific serialization CHIP instead